cardano-testnet/test/cardano-testnet-test/cardano-testnet-test.hs

```diff
@@ -50,7 +49,8 @@ tests = do
             -- TODO: Replace foldBlocks with checkLedgerStateCondition
             , T.testGroup "Governance"
                 [ ignoreOnMacAndWindows "Committee Add New" Gov.hprop_constitutional_committee_add_new
-                , ignoreOnWindows "DRep Activity" Gov.hprop_check_drep_activity
+                -- FIXME: This test is broken - drepActivity is not updated within the expeted period
+                -- , ignoreOnWindows "DRep Activity" Gov.hprop_check_drep_activity
```

Comment:
@Lucsanszky @carbolymer Could you elaborate on this?

Reply:
I was not able to quickly fix this, needs deeper investigation why drep activity isn't updated: https://input-output-rnd.slack.com/archives/C031GFLC3DF/p1715775093711539?thread_ts=1715766601.135709&cid=C031GFLC3DF

Reply:
Thanks :+1: